client/app/src/main/java/com/microsoft/research/karya/ui/registration/RegistrationActivity.kt

```diff
@@ -0,0 +1,54 @@
+package com.microsoft.research.karya.ui.registration
+
+import android.os.Bundle
+import android.util.Log
+import com.microsoft.research.karya.R
+import com.microsoft.research.karya.ui.base.BaseActivity
+
+class RegistrationActivity : BaseActivity(useAssistant = true) {
```

Comment:
Don't inherit from `BaseActivity` since we are planning to remove it.

Reply:
Sure